packages/xcb_proto.rb

```diff
@@ -0,0 +1,20 @@
+require 'package'
+
+class Xcb_proto < Package
+  description 'The protocols for the X window system provide extended functionality for communication between a X client and the server.'
+  homepage 'https://x.org'
+  version '1.12'
+  source_url 'https://www.x.org/archive/individual/xcb/xcb-proto-1.12.tar.gz'
+  source_sha256 'cfa49e65dd390233d560ce4476575e4b76e505a0e0bacdfb5ba6f8d0af53fd59'
+
+  depends_on 'python'
```

Comment:
Shouldn't this be changed to python27?
When running the ./configure, it found python with version 2.7
```
checking for python... /usr/local/bin/python
checking for python version... 2.7
```
(and the 'python' package installs python3, so this won't influence it, or maybe it accepts both)

I'm not sure, python is confusing (#923 is relevant)